spec/lib/whitehall_importer/create_migration_spec.rb

```diff
@@ -36,6 +36,11 @@
           .to change { WhitehallMigration::DocumentImport.pending.count }.by(110)
       end
 
+      it "sets the content_id on the WhitehallMigration::DocumentImport" do
+        described_class.call("123", "news_article")
+        expect(WhitehallMigration::DocumentImport.last.content_id).not_to eq(nil)
```

Comment:
Could this be updated to check the correct ID is set, e.g.
```
expect(WhitehallMigration::DocumentImport.last.content_id).to eq(whitehall_export_page_1.last[:content_id])
```

Reply:
👍